code/modules/mob/living/carbon/life.dm

```diff
@@ -55,6 +55,7 @@
 	if(knocked_out || sleeping || getOxyLoss() > CARBON_KO_OXYLOSS || health < get_crit_threshold())
 		if(stat != UNCONSCIOUS)
 			blind_eyes(1)
+			DEAF(1<<2)
```

Comment:
```suggestion
			M.disabilities |= DEAF
```

`M.disabilities` is a bitfield, which can store multiple types of bit shifted values (things assigned like `1<<2`).
When you do `|=` This is like saying add `DEAF` to `M.disabilties`, but in a bitfield way.